packages/upward-js/bin/server

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+require('dotenv').config();
+
+function castValue(value) {
+    const mapping = {
+        false: false,
+        true: true
+    };
```

Comment:
Consider using a `Map` for this kind of thing.

Reply:
It was so small, I thought it okay this once.